cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -60,7 +60,7 @@ allowPrivilegedContainer: true
 allowHostNetwork: true
 allowHostDirVolumePlugin: true
 priority:
-allowedCapabilities: ['*']
+allowedCapabilities: ["*"]
```

Comment:
Why is this change needed? Was there some error?

Reply:
No, I dont know why this got changed.
My VScode sometime acts strange.

Reply:
Done!, please review now, I have fixed all this.
thanks